Yosemite/Yosemite/Model/Storage/OrderCount+ReadOnlyConvertible.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Storage
+
+
+// MARK: - Storage.OrderCount: ReadOnlyConvertible
+//
+extension Storage.OrderCount: ReadOnlyConvertible {
+
+    /// Updates the Storage.OrderCount with the ReadOnly.
+    ///
+    public func update(with orderCount: Yosemite.OrderCount) {
+        siteID = Int64(orderCount.siteID)
```

Comment:
Ideally we would, and I tried. However, I could not make it work. I am not completely sure why, but I think it's because we are going in and out of the boundary between Yosemite and Network, trying to assign types from one side to properties typed in the other.

I looked around, and noticed that those relationships are assigned in the Stores elsewhere, and went with the as well. I do agree though that the relationship should be assigned within this method.

I am going to defer this for now as well, if you don't mind, we can come back to it later if necessary?

Reply:
yup sounds good!